modules/web-mercator/docs/api-reference/web-mercator-viewport.md

```diff
@@ -91,6 +91,19 @@ Returns:
  - `[longitude, latitude]`
 
 
+##### `getBounds(options)`
+
+Returns the bounding box of the current visible area.
+
+* `options` (Object, optional)
+  + `options.z` (Number, optional) - The altitude at which the bounds should be calculated. Default `0`.
+  + `options.quad` (Boolean, optional) - If `pitch` or `bearing` is not zero, the visible region is not an axis-aligned rectangle. This option controls the format of the returned bounding box, see below. Default `false`.
+
+Returns:
+* By default: `[[lon, lat], [lon, lat]]` as the south west and north east corners of the smallest orthogonal bounds that encompasses the visible region.
```

Comment:
Maybe two methods is better than a one method that returns different things? 

- `getBoundingRectangle()`
- `getBoundingRegion()` 

I assume the region is "tighter" but more complex to use, and the rectangle is the smallest axis aligned rectangle that fits the region.

Reply:
I can separate the two. However the rectangular flavor is the same as mapbox's `getBounds()`. I prefer to not invent a new name there.

Reply:
Sure: `getBounds` and `getBoundsAsRegion()`?